README.md

```diff
@@ -15,25 +15,27 @@ thing! https://github.com/PolymerLabs/tedium/issues
 [![Build status](https://travis-ci.org/PolymerElements/marked-element.svg?branch=master)](https://travis-ci.org/PolymerElements/marked-element)
 
 
-##&lt;marked-element&gt;
+## &lt;marked-element&gt;
 
 Element wrapper for the [marked](https://github.com/chjj/marked) library.
 
-`<marked-element>` accepts Markdown source, and renders it to a child
+`<marked-element>` accepts Markdown source and renders it to a child
 element with the class `markdown-html`. This child element can be styled
 as you would a normal DOM element. If you do not provide a child element
 with the `markdown-html` class, the Markdown source will still be rendered,
 but to a shadow DOM child that cannot be styled.
 
-The Markdown source can be specified either via the `markdown` attribute:
+The Markdown source can be specified several ways:
+
+via the `markdown` attribute
```

Comment:
For each of these different ways to specify source, turn this into a `###` (cleaning up the wording to be more header-like might be needed as well)